src/app.py

```diff
@@ -1,32 +1,58 @@
+import logging
+import sys  # Import sys to access stdout
 from flask import Flask
-from src.routes.db_access import db_test_blueprint
+from routes import quote_bp
 from flask_cors import CORS
-import os
-from src.models import models
+import models
+from config import Config  # Import the Config class
 
+logger = logging.getLogger(__name__)
+# Configure logging to stdout
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG if Config.FLASK_ENV == 'development' else logging.INFO)  # Set logging level based on environment
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(formatter)
+
+logger.setLevel(logging.INFO)
+logger.addHandler(handler)
 
 def create_app():
     app = Flask(__name__)
 
     app.config["SQLALCHEMY_DATABASE_URI"] = (
         f"mysql+pymysql://"
-        f"{os.environ.get('MYSQL_USER')}:{os.environ.get('MYSQL_PASSWORD')}"
-        f"@{os.environ.get('MYSQL_HOST')}/{os.environ.get('MYSQL_DATABASE')}"
+        f"{Config.MYSQL_USER}:{Config.MYSQL_PASSWORD}"
+        f"@{Config.MYSQL_HOST}/{Config.MYSQL_DATABASE}"
```

Comment:
@PedroATCosta and @iiscas we fetch the configs from the `config.py`. 
Otherwise, if we fetch them from the env vars directly, the `config.py` becomes unecessary.